src/openeo_gfmap/manager/job_manager.py

```diff
@@ -83,16 +121,30 @@ def __init__(
 
         self._root_collection = self._normalize_stac()
 
+        # Add a property that calculates the number of maximum concurrent jobs
+        # dinamically depending on the time
+        self._dynamic_max_jobs = dynamic_max_jobs
+        self._max_jobs_worktime = max_jobs_worktime
+        self._max_jobs = max_jobs
+
     def _normalize_stac(self):
```

Comment:
can you include a docstring?

Reply:
In addition I do not think the name of the function expresses wat it does. 

initiate_stac_collection ? or something amongst those lines

Reply:
in addition For future testing I would make the function more modular. Now it does more than 1 thing:

it:
-checks
-loads
and creates

I would split these 3 actions up in a more modular way